README.md

```diff
@@ -76,6 +76,7 @@ API | Description | Auth | HTTPS | CORS |
 | [RandomFox](https://randomfox.ca/floof/) | Random pictures of foxes | No | Yes | No |
 | [RescueGroups](https://userguide.rescuegroups.org/display/APIDG/API+Developers+Guide+Home) | Adoption | No | Yes | Unknown |
 | [Shibe.Online](http://shibe.online/) | Random pictures of Shibu Inu, cats or birds | No | No | No |
+| [Slash the API Dog](https://slashtheapidog.com) | API for Slash's bones, balls, and holes | No | Yes | No |
```

Comment:
Seems like this is a testdata API - can you confirm?

Reply:
Also, please add a more descriptive Description for the purpose/use-case of this service.